prov/efa/src/rdm/efa_rdm_srx.c

```diff
@@ -228,6 +228,7 @@ static int efa_rdm_srx_queue_tag(struct fi_peer_rx_entry *peer_rx_entry)
 	ep = rx_entry->ep;
 
 	ofi_mutex_lock(&ep->base_ep.util_ep.lock);
+	rx_entry->total_len = rx_entry->peer_rx_entry.size;
```

Comment:
Is this unique to tagged message? Do we need to, or does it hurt to do the same on L208 for untagged message?

Reply:
>Is this unique to tagged message?

Yes, this is unique to tagged message because of 2 reasons.

1. FI_PEEK only applies to tagged message
2. For the non-tagged case, the size of rx_entry is already updated when peer provider calls `get_msg`, as the msg size is the input of this API call.

Reply:
Actually, this change can be avoided if we can combine the duplicate fields in rx_entry and rx_entry.peer_rx_entry. I will add a TODO here.